packages/plugins/rum/src/built/rum-browser-sdk.ts

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+/* global globalThis */
+
+import { datadogRum } from '@datadog/browser-rum';
```

Comment:
Shouldn't you add this to your peerDeps too?

Because otherwise customers won't be able to import this as this is in our devDeps

Reply:
Also you may want to mark this as optional peerDep (for people using `DD_RUM` and not the npm package)

and if it's an optional peerDep, the import could fail. So you need to do
```js
let datadogRum = {}
try {
 datadogRum = require('@datadog/browser-rum').datadogRum
} catch {}
```

Reply:
I'll add a comment, but this is bundled outside of the plugin.
Only used in case of injection.

In short, when I bundle the plugin, I also bundle this file on the side.
And if the customer wants to inject the SDK, I add the content of that bundled file in the `MIDDLE` injection point.

Reply:
> Only used in case of injection.

But in case of injection, **they** need to be able to import `@datadog/browser-rum`.  So it needs to be in **their** dependencies (thus the peerDep)

Reply:
> And if the customer wants to inject the SDK

This is also why I recommended an optional peerDep
Maybe we don't need the try / catch though, you're right

Reply:
It's not an external dependency, `@datadog/browser-rum` is already bundled in the injected code.

Reply:
AH, it's fully pre-bundled, I see. I may need to revise other comments too then 👌